src/hotspot/os_cpu/linux_aarch64/atomic_linux_aarch64.hpp

```diff
@@ -27,57 +27,184 @@
 #define OS_CPU_LINUX_AARCH64_ATOMIC_LINUX_AARCH64_HPP
 
 #include "runtime/vm_version.hpp"
+#include "atomic_aarch64.hpp"
```

Comment:
Should be sorted before `#include "runtime/vm_version.hpp"`

Reply:
Thanks.